src/OpenSpaceToolkit/Mathematics/Object/Interval.tpp

```diff
@@ -98,42 +123,32 @@ bool Interval<T>::contains(const T& aValue) const
         throw ostk::core::error::runtime::Undefined("Interval");
     }
 
-    switch (type_)
-    {
-        case Interval<T>::Type::Closed:
-            return (lowerBound_ <= aValue) && (aValue <= upperBound_);
-
-        case Interval<T>::Type::Open:
-            return (lowerBound_ < aValue) && (aValue < upperBound_);
-
-        case Interval<T>::Type::HalfOpenLeft:
-            return (lowerBound_ < aValue) && (aValue <= upperBound_);
-
-        case Interval<T>::Type::HalfOpenRight:
-            return (lowerBound_ <= aValue) && (aValue < upperBound_);
-
-        default:
-            throw ostk::core::error::runtime::Wrong("Type");
-            break;
-    }
-
-    return false;
+    return this->contains(aValue, false);
```

Comment:
Should this always be false? or should this be based on if this->type_ == closed or not?

Reply:
The `false` refers to the value itself (in case we're checking a boundary), but since it's a scalar value `false` is fine.

The type_ == closed is considered within the checkAgainsLower/Upper function.